### PR TITLE
#1178 remove letter and pdf

### PR DESCRIPTION
**Pull request by babyjunior56**
# Description

Removed all code that used:
Celery tasks - app/celery/letters_pdf_tasks.py
Removed all code that required the creation or uploading  letters
Removed all code that required the pdf endpoint
Removed all code that required the letters endpont
Removed unused variables
   

issue #1178 

## Type of change

Please check the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation changes only

## How Has This Been Tested?
- [x] Ran Unit test locally
- [x] Deployed to Dev 

## Checklist

- [x] I have assigned myself to this PR
- [x] PR has an appropriate title: #9999 - What the thing does
- [x] PR has a detailed description, including links to specific documentation
- [x] I have added the [appropriate labels](https://github.com/department-of-veterans-affairs/notification-api/blob/master/.github/release.yaml) to the PR.
- [x] I did not remove any parts of the template, such as checkboxes even if they are not used
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to any documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have ensured the latest master is merged into my branch and all checks are green prior to review
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] The ticket is now moved into the DEV test column

### CODE REVIEW

**Review by kalbfled — CHANGES_REQUESTED** [review]
You have not deleted [this](https://github.com/department-of-veterans-affairs/notification-api/blob/master/app/v2/notifications/get_notifications.py#L30) route.

File: app/celery/tasks.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     notify_celery,
 )
 from app.aws import s3
-from app.celery import provider_tasks, letters_pdf_tasks, research_mode_tasks
+from app.celery import provider_tasks, research_mode_tasks
 from app.config import QueueNames
 from app.dao.daily_sorted_letter_dao import dao_create_or_update_daily_sorted_letter
```
Comment (cris-oddball) [inline app/celery/tasks.py:25]: Shouldn't this import be removed? `dao_create_or_update_daily_sorted_letter`

Comment (cris-oddball) [reply app/celery/tasks.py:25]: It's used once later in this file starting at line 460 in `def persist_daily_sorted_letter_counts()` - shouldn't that also be removed?

Comment (cris-oddball) [reply app/celery/tasks.py:25]: There are in fact other letters methods in this file - why are we keeping those?

File: app/v2/notifications/post_notifications.py [hunk 2]
```diff
@@ -1,37 +1,20 @@
...
-from app.dao.templates_dao import get_precompiled_letter_template
+from app.config import QueueNames
 from app.feature_flags import accept_recipient_identifiers_enabled, is_feature_enabled, FeatureFlag
-from app.letters.utils import upload_letter_pdf
 from app.models import (
     SCHEDULE_NOTIFICATIONS,
     SMS_TYPE,
     EMAIL_TYPE,
     LETTER_TYPE,
     UPLOAD_DOCUMENT,
```
Comment (cris-oddball) [inline app/v2/notifications/post_notifications.py:16]: Why are we importing these?

File: app/v2/notifications/post_notifications.py [hunk 3]
```diff
@@ -59,52 +42,10 @@
     post_sms_request,
     post_email_request,
     post_letter_request,
```
Comment (cris-oddball) [inline app/v2/notifications/post_notifications.py:44]: why are we importing this?

File: tests/app/v2/notifications/test_post_notifications.py [hunk 4]
```diff
@@ -952,7 +951,6 @@ def test_attachment_upload_unsupported_mimetype(
             self,
             client,
             notify_db_session,
-            mocker,
             service_with_upload_document_permission,
```
Comment (cris-oddball) [inline tests/app/v2/notifications/test_post_notifications.py:954]: is this used?

**Review by cris-oddball — CHANGES_REQUESTED** [review]
Seems like there are still a lot of letters related items that in these files?

**Pull request closed without merging**